frameworks/cassandra/tests/test_sanity.py

```diff
@@ -28,14 +28,12 @@ def configure_package(configure_universe):
         sdk_install.uninstall(FOLDERED_SERVICE_NAME, package_name=PACKAGE_NAME)
         sdk_utils.gc_frameworks()
 
-        # check_suppression=False due to https://jira.mesosphere.com/browse/CASSANDRA-568
-        sdk_install.install(
+        sdk_upgrade.test_upgrade(
+            "beta-{}".format(PACKAGE_NAME),
             PACKAGE_NAME,
             DEFAULT_TASK_COUNT,
             service_name=FOLDERED_SERVICE_NAME,
-            additional_options={"service": { "name": FOLDERED_SERVICE_NAME } },
-            check_suppression=False)
-        sdk_plan.wait_for_completed_deployment(FOLDERED_SERVICE_NAME)
+            additional_options={"service": {"name": FOLDERED_SERVICE_NAME} })
```

Comment:
Looks like this is a problem with Kafka. I have a Kafka test fix that I'll be pushing in a bit.

Cassandra meanwhile is already defaulting to user=nobody in the latest universe release. Cassandra has a different issue with volumes moving from tasks to executors.